blog/2024-12-03-frontend-decorator-pattern.md

```
@@ -0,0 +1,458 @@
+    // Log error message
+  }
+}
+
+const logger = new Logger();
+logger.info(); // Logs "[INFO] info called"
+logger.error(); // Logs "[ERROR] error called"
+```
+
+위의 코드에서 사용자 정의 데코레이터는 `Logger` 클래스 내의 메서드를 강화하는 데 사용된다. 이러한 데코레이터는 
+
+https://www.sitepoint.com/javascript-decorators-what-they-are/#whataredecoratorsinjavascript
```

Markdownlint (0.35.0)

Bare URL used

(MD034, no-bare-urls)